x-pack/plugins/spaces/server/lib/request_interceptors/integration_tests/on_post_auth_interceptor.test.ts

```diff
@@ -41,74 +38,25 @@ describe.skip('onPostAuthInterceptor', () => {
     ).toString('base64')}`,
   };
 
-  /**
-   *
-   * commented out due to hooks being called regardless of skip
-   * https://github.com/facebook/jest/issues/8379
-
   beforeEach(async () => {
     root = kbnTestServer.createRoot();
   });
 
   afterEach(async () => await root.shutdown());
 
-  */
```

Comment:
These tests have been disabled for some time due to flakyness. Since we are mostly done with the legacy platform, I opted to remove the LP tests from this suite in order to simplify and reduce the chance of further flakyness. Tests for only the Kibana Platform are better than no tests at all.